release_docs/USING_CMake_Examples.txt

```diff
@@ -76,19 +130,142 @@ Default installation process:
     add "-VV" to the ctest command. The output should show;
           100% tests passed, 0 tests failed out of 206.
 
+  3. Using Command Line CMake
+
+    A. Visual Configuration
+
+      The visual CMake executable is named "cmake-gui.exe" on Windows and should be
+      available in your Start menu. For Linux, UNIX, and Mac users the
+      executable is named "cmake-gui" or the ncurses-based "ccmake" and can be found
+      where CMake was installed.
+
+      Specify the source and build directories. Make the build and source
+      directories different. For example on Windows, if the source is at
+        C:\MyHDFstuff\hdf5ex, then use C:\MyHDFstuff\hdf5ex\build or
+        C:\MyHDFstuff\build\hdf5ex for the build directory.
+
+      Click the Configure button. If this is the first time you are
+      running cmake-gui in this directory, you will be prompted for the
+      generator you wish to use (for example on Windows, Visual Studio 15).
+      CMake will read in the CMakeLists.txt files from the source directory and
+      display options for the HDF5 Examples project. After the first configure you
+      can adjust the cache settings and/or specify locations of other programs.
+
+      Any conflicts or new values will be highlighted by the configure
+      process in red. Once you are happy with all the settings and there are no
+      more values in red, click the Generate button to produce the appropriate
+      build files.
+
+      On Windows, if you are using a Visual Studio generator, the solution and
+      project files will be created in the build folder.
+
+      On linux, if you are using the Unix Makefiles generator, the Makefiles will
+      be created in the build folder.
+
+    B. Alternative Command Line Configuration
+
+      Users can perform the configuration step without using the visual
+      cmake-gui program. The following is an example command line
+      configuration step executed within the build directory:
+
+        cmake -G "<generator>"  [-D<options>]  <sourcepath>
+
+        Where <generator> is
+            * MSYS Makefiles
+            * MinGW Makefiles
+            * NMake Makefiles
+            * Unix Makefiles
+            * Visual Studio 15 2017
+            * Visual Studio 15 2017 Win64
+            * Visual Studio 16 2019
+            * ... in addition VS2019 will need to set the "-A" option,
+            * ... [Win32, x64, ARM, ARM64]
+            * Visual Studio 17 2022
+            * ... in addition VS2022 will need to set the "-A" option,
+            * ... [Win32, x64, ARM, ARM64]
+
```

Comment:
What about Ninja?

This, too, will need to be constantly updated. Can you think of an alternative way to present this that doesn't add to cognitive debt?

Reply:
Added "(examples)" to indicate it is partial list. Could add "partial" or "incomplete" as well?